model/host/db.go

```diff
@@ -587,13 +587,14 @@ func ById(id string) db.Q {
 	return db.Query(bson.D{{Key: IdKey, Value: id}})
 }
 
-// ByIP produces a query that returns a host with the given ip address.
-func ByIP(ip string) db.Q {
+// ByIP produces a query that returns a running host with the given ip address.
```

Comment:
don't forget the function name in its comment!

Reply:
ahh sorry! done